types.d.ts

```diff
@@ -0,0 +1,691 @@
+/**
```

Comment:
Maybe this should be added to `.gitignore`?

Reply:
Uhm... Could be 🤔. I think it's nice to have it as a quick go-to readme, though. We could maybe add a banner at the top of the file saying it was "auto-generated" and that it "should not be modified by hand"?

Reply:
Added banner to `index.d.ts` (former `types.d.ts`).